chip8-dis/src/main.rs

```diff
@@ -0,0 +1,28 @@
+mod chip8;
+
+use chip8::ops::Op;
+use std::env;
+use std::fs::File;
+use std::io::Read;
+
+fn main() {
+    let file = env::args().skip(1)
+                          .nth(0)
+                          .and_then(|path| File::open(path).ok());
+
+    if file.is_none() {
+        eprintln!("Missing expected file path as first argument or the specified file does not exist.");
+        return;
+    }
+    
+    let file = file.unwrap();
+
+    file.bytes()
+        .filter_map(Result::ok)
+        .map(|b| (b as u16).swap_bytes())
```

Comment:
I'm aware of the issue here. I couldn't determine a proper solution to this without pulling in a 3rd party crate, such as `byteorder`. I decided that this was good enough for a rough draft translation from C to Rust.